geth/account/importer.go

```diff
@@ -0,0 +1,34 @@
+package account
+
+import (
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/status-im/status-go/extkeys"
+)
+
+// extendedKeyImporter import ECDSA key (obtained from extended key) and returns decrypted key for account.
+type extendedKeyImporter interface {
+	Import(keyStore accountKeyStorer, extKey *extkeys.ExtendedKey, password string) (address, pubKey string, err error)
+}
+
+type extendedKeyImporterBase struct{}
```

Comment:
See question above about naming `...Base`.

Reply:
Ok, i think i should change it.